docs/blog/2024-05-08-nx-19-release.md

```diff
@@ -0,0 +1,233 @@
+
+Going forward, these environment variables will need to be prefixed with `NX_PUBLIC_`. This is a security consideration - by requiring a more specific prefix we're making sure that no secrets end up in your production builds unintentionally!
+
+Because of the nature of this change we will NOT be providing an automatic migration. So be sure to manually update any environment variables that you want bundled into your builds from `NX_` to `NX_PUBLIC_`.
+
+## General Crystal Polishing
+
+Along with the features above (many of which further build upon Nx Project Crystal), we've done quite a bit since Nx 18 to polish the features that we introduced with Project Crystal.
+
+Since Nx 18 release, we also started using Project Crystal inside of the Nx repo itself (as well as in Nx Console and in our closed-source monorepo as well). This has allowed us to find many improvements we can make, as well as issues we can fix.
+
+You can find a full list of fixes and features applied in this major release [here](https://github.com/nrwl/nx/releases/tag/19.0.0).
+
+[![Changelog for Nx 19](/blog/images/2024-05-08/fixes.gif)](https://github.com/nrwl/nx/releases/tag/19.0.0)
+
+With Project Crystal landed now, we're also adjusting our priorities to place a higher importance on stability. You should see this reflected in Nx 19.
+
+As a note, we had noticed an increase in the rate of Github issues since Nx 18 launched. Previously, we had not allocated enough resources to handling these issues. To address this, our team is going to be doubling the amount of time focusing on maintainence.
+
+In addition - we are adding more process to make the triage of these issues more efficient. Github issues will now be assigned directly to specific members of the Nx Core Team. The main intention of this change is to increase the efficiency of the resources that we do spend on addressing issues.
+
+## Nx Cloud Updates!
+
+[![The Nx Cloud Dashboard](/blog/images/2024-05-08/nx-cloud-dashboard.webp)](https://nx.app/#deep-understanding)
+
+We've got some cool statics to share from our users regarding the benefits of [Nx Cloud](https://nx.app), our premium CI service. The three areas we've identified as the critical aspects of a CI provider are: speed, cost, and reliablity. In these areas we've seen:
+
+- **speed**: Reported 30% - 70% faster CI
+- **cost**: Reported 40% - 75% reduction in CI costs
+- **reliability**: This is a more difficult one to quantify! But Nx Cloud's automatic detection and retrying of flaky tests has been super helpful for them!
```

Comment:
You can link to this post:
https://nx.dev/blog/2024-03-21-reliable-ci

It's quantifiable. The numbers, though, are a lot sharper. Basically the issue with flaky tests largely goes away completely.